frontend/src/employee-frontend/components/vasu/VasuEditPage.tsx

```diff
@@ -151,12 +156,17 @@ export default React.memo(function VasuEditPage({
             {showSpinner && <Spinner />}
           </StatusContainer>
           {vasu && (
-            <Button
-              text={i18n.vasu.checkInPreview}
-              disabled={status.state != 'clean'}
-              onClick={() => history.push(`/vasu/${vasu.id}`)}
-              primary
-            />
+            <FullWidthDiv>
+              <ButtonContainer>
+                <Button
+                  text={i18n.vasu.checkInPreview}
+                  disabled={status.state != 'clean'}
+                  onClick={() => history.push(`/vasu/${vasu.id}`)}
+                  primary
+                />
+                <LeaveVasuPageButton childId={vasu.basics.child.id} />
```

Comment:
Should this be disabled too if state is not clean?

Reply:
Good catch! Fixed.